schedule.py

```diff
@@ -0,0 +1,47 @@
+from typing import Iterable
+from flask_restful import abort
+
+from scheduledao import ScheduleDAO
+from event import Event
+
+class Schedule:
+    dao = ScheduleDAO()
+
+    def __init__(self, document: dict):
+        self.id = str(document['_id']) if '_id' in document else None
+        self.owner = document['owner'] if 'owner' in document else None
+        self.events = document['events'] if 'events' in document else []
+
+    @classmethod
+    def get(cls, owner=None):
+        sched_dicts = cls.dao.get(owner)
+        if not isinstance(sched_dicts, list):
```

Comment:
you know what the DAO will return, it SHOULD only ever return a `list` or `None` just check that it is not None ... isinstance is much more expensive than `if sched_dicts:`